src/judges/codeforces/getContests.js

```diff
@@ -1,46 +1,39 @@
 // @flow
-import request from 'request';
+import request from 'request-promise-native';
 import type { Contest } from '../../types';
 
-export function getContests(): Promise<Array<Contest>> {
-  return new Promise((resolve, reject) => {
-    // making request to the codeforces api
-    request('http://codeforces.com/api/contest.list', (error: Object, response: Object, body: string) => {
-      if (response && response.statusCode == 200) {
-        let contests: Array<Contest> = [];
+export async function getContests(): Promise<Array<Contest>> {
+  const body = await request('http://codeforces.com/api/contest.list');
+  const contests: Array<Contest> = [];
 
-        let data: Object = JSON.parse(body).result;
-        for (let index: string in data) {
-          let info: Object = data[index];
+  const data: Object = JSON.parse(body).result;
+  for (let index: string in data) {
+    const info: Object = data[index];
```

Comment:
I don't like `info`. Maybe `cfContest` or `codeforcesContest`. I don't know, but again, `info` is way too generic.

Reply:
Those names aren't from my PR, but since I'm messing with this I'll change them.

Reply:
Again, I have the same opinion as on `data`.